packages/web-components/fast-components/src/radio-group/radio-group.styles.ts

```diff
@@ -0,0 +1,54 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "../styles";
+import { focusVisible } from "../styles/focus";
+import { SystemColors } from "../styles/system-colors";
+import { heightNumber } from "../styles/size";
+
+export const RadioGroupStyles = css`
+    ${display("inline-flex")} :host {
+        --input-size: calc((${heightNumber} / 2) + var(--design-unit));
+        align-items: center;
+        outline: none;
+        margin: calc(var(--design-unit) * 1px) 0;
+        ${
+            /*
+             * Chromium likes to select label text or the default slot when
+             * the radio button is clicked. Maybe there is a better solution here?
+             */ ""
+        } user-select: none;
+        position: relative;
+        flex-direction: row;
+        transition: all 0.2s ease-in-out;
+    }
+
+    .control {
+        position: relative;
+        box-sizing: border-box;
+        outline: none;
+        cursor: pointer;
+    }
+
+    :host(:${focusVisible}) .control {
+        box-shadow: 0 0 0 1px var(--neutral-focus) inset;
+        border-color: var(--neutral-focus);
+    }
+
+    :host(.disabled) {
+        opacity: var(--disabled-opacity);
+    }
+
+    @media (forced-colors: active) {  
```

Comment:
Check with @nicholasrice since we've got a new way to conditionally include the high contrast styles.

Reply:
oh yea I saw that on tabs, I'll change it to be like that